spec/lib/va1010_forms/enrollment_system/service_spec.rb

```diff
@@ -0,0 +1,153 @@
+# frozen_string_literal: true
```

Comment:
Could it make sense to break this PR up into two PRs? I'm thinking one to add the new `VA1010Forms::EnrollmentSystem::Service` and it's corresponding specs in one, and then a second one to add the flipper and actually call the new service in the implementation? That may help us isolate the new behavior better and make a review by platform easier.